packages/apps/composer-app/src/components/Placeholder.tsx

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React, { useEffect, useState } from 'react';
+
+import { DXOS } from '@dxos/brand';
+import { Status, ThemeProvider } from '@dxos/react-ui';
+import { defaultTx, getSize, mx } from '@dxos/react-ui-theme';
+
+export const Placeholder = () => {
```

Comment:
The root node of this is `Status`, so the name of this component should reflect that. `Loader` is also a worthwhile term here, but `Placeholder` signals (for me) that something did not load or that this is a design prototype.